packages/glimmer-runtime/lib/syntax/core.ts

```diff
@@ -544,8 +563,13 @@ export class OpenElement extends StatementSyntax {
     let current = scanner.next();
     let args = dict<ExpressionSyntax<Opaque>>();
     let attrs: InternedString[] = [];
+    let isModifier = !!current[MODIFIER_SYNTAX];
```

Comment:
I don't think we can cache this – if we have `<foo-bar bat="baz" {{modifier}} />` the first thing we see would be an attribute syntax (bat="baz"), followed by a modifier. In this case, because we never updated `isModifier` inside the loop, it will always be false. (Probably don't need the `!!` also)